projects/packages/jetpack-mu-wpcom/src/features/wpcom-wpadmin-page-view/wpcom-wpadmin-page-view.php

```diff
@@ -0,0 +1,188 @@
+		return wpcom_track_customizer_from_frontend();
+	}
+
+	// standard wp-admin
+	wpcom_nosara_track_admin_page_views();
+}
+
+/**
+ * Since links to the Customizer from the frontend aren't properly wp-admin (no wp-admin chrome,
+ * you return to the frontend after closing), they shouldn't be counted as wp-admin views.
+ */
+function wpcom_track_customizer_from_frontend() {
+	global $current_user;
+
+	// Since we view so many screens to offer support and do feature dev, I think it's best not to count us.
+	if ( do_not_track_a11ns() ) {
+		return;
+	}
+	?>
+	<script type="text/javascript">
+		_tkq = window._tkq || [];
+		_tkq.push( [ 'identifyUser', <?php echo (int) $current_user->ID; ?>, '<?php echo esc_js( $current_user->user_login ); ?>' ] );
+		_tkq.push( [ 'recordEvent', 'wpcom_customize_loaded_from_frontend' ] );
+	</script>
+	<?php
+}
+add_action( 'customize_controls_print_footer_scripts', __NAMESPACE__ . '\wpcom_maybe_track_customizer' );
+
+/**
+ * Determine if the current user should not be tracked.
+ *
+ * @return bool
+ */
+function do_not_track_a11ns() {
+	$is_simple_site = defined( 'IS_WPCOM' ) && IS_WPCOM;
+	$is_atomic_site = ! $is_simple_site;
+
+	if ( $is_simple_site ) {
+		// Since we view so many screens to offer support and do feature dev, I think it's best not to count us.
+		if ( is_automattician() ) {
+			return true;
+		}
+
+		// Shouldn't hit this because of the is_automattician() check above, but let's be cautious.
+		if ( is_network_admin() ) {
+			return true;
+		}
+	}
+
+	if ( $is_atomic_site ) {
+		return wpcom_atomic_maybe_is_a11n();
+	}
+}
+
+/**
+ * Check if the user might be an a11n on Atomic sites.
+ *
+ * @return bool
+ */
+function wpcom_atomic_maybe_is_a11n() {
+	$is_proxy_atomic    = defined( 'AT_PROXIED_REQUEST' ) && AT_PROXIED_REQUEST;
+	$is_support_session = WPCOMSH_Support_Session_Detect::is_probably_support_session();
+
+	return $is_proxy_atomic && ! $is_support_session;
+}
+
+/**
+ * Retrieves the user types for Atomic sites.
+ *
+ * @return array An array of user types.
+ */
+function wpcom_atomic_get_user_types() {
+	$user_types = array();
+
+	if ( get_account_age_in_days() <= 14 ) {
+		$user_types[] = 'New User';
+	}
+
+	// @phan-suppress-next-line PhanUndeclaredClassMethod -- This class is available in atomic sites.
+	$purchases = \Automattic\Jetpack\VideoPress\Site::get_purchases();
+	if ( count( $purchases ) > 0 ) {
+		$user_types[] = 'Paid';
+	}
+
+	if ( wpcom_site_has_feature( \WPCOM_Features::PRIORITY_SUPPORT ) ) {
+		$user_types[] = 'Business';
+	}
+
+	return $user_types;
+}
+
+/**
+ * Calculate the number of days since the current user registered their account.
+ *
+ * @global WP_User $current_user The current user object.
+ *
+ * @return float The age of the account in days.
+ */
+function get_account_age_in_days() {
+	global $current_user;
+
+	return ( time() - strtotime( $current_user->user_registered ) ) / DAY_IN_SECONDS;
```

Comment:
I'm not sure if this is true. Atomic is on a different infrastructure, meaning it uses a different db table for users. Do we send the user_registered from the WPCOM user when a site goes on Atomic?

You can check when your WPCOM simple user was registered and compare the date with a user from an Atomic site.

Reply:
I do get the same date. On wp-admin I have this:

![image](https://github.com/user-attachments/assets/bb099be7-bd87-4bfb-adea-f5fbbe82d54b)

And dumping `$current_user->user_registered` returns `"2008-02-29 23:42:01"`.